edxval/api.py

```diff
@@ -158,6 +158,33 @@ def update_video(video_data):
         raise ValCannotUpdateError(serializer.errors)
 
 
+def update_video_status(video_data):
```

Comment:
Nit: Per clean-code best practices, it's better to send only the data that this function needs as separate parameters.  So `video_id` and `status` should be separate parameters to this function.  And then, the implementation of this method need not be as defensive - that is, no longer a need to call `get` on the given `dict`.